ci/workflows/main-cron.yml

```diff
@@ -483,6 +483,23 @@ steps:
     timeout_in_minutes: 25
     retry: *auto-retry
 
+  - label: "GCS source check on AWS (json parser)"
```

Comment:
The test passed in gcp machine but fail in out CI, I guess it's because the gateway/network issue while doing credential check, so I suggest only leave s3 source in CI. 
<img width="1092" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/58715567/6b0766ee-2168-4c14-9b00-0266a2165756">

Reply:
@huangjw806 any solution for testing GCS?

Reply:
gcloud auth activate-service-account 

Maybe you need to auth gcloud with service account key.